k8s/README.md

```diff
@@ -0,0 +1,400 @@
+# linkerd on Kubernetes #
+
+This doc explains how to get Gob's Microservice up and running 
+
+This example should work in any kubernetes cluster.  If you don't have one at
+your disposal, though, we can walk you through setting one up:
+
```

Comment:
it might be good to call out what prior knowledge we assume the reader should already have.  I think we're assuming the reader is already familiar with k8s concepts and terminology, as well as what linkerd is and what it does.